docs/deployment-examples/terraform-deployments/remote-state/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Terraform Remote State
+description: Using remote state with Octopus
+---
+
+When running Terraform on a local PC, the state of the resources managed by Terraform is saved in a local file. This state is queried to learn which resources already exist in order to properly apply updates and destroy resources.
+
+When Terraform is run by Octopus, this state file is not preserved between executions. This means that, for almost all practical applications of Terraform through Octopus, a remote backend must be configured. A remote backend allows this state information to be preserved between Terraform steps.
```

Comment:
Slight tweak to change the order of the sentence and avoid the parenthetical phrase:
```suggestion
When Terraform is run by Octopus, this state file is not preserved between executions. This means a remote backend must be configured for almost all practical applications of Terraform through Octopus, allowing the state information to be preserved between Terraform steps.
```